src/core/generators/angular.ts

```diff
@@ -53,14 +54,15 @@ ${
   headers?: HttpHeaders | {
       [header: string]: string | string[];
   };
-  observe?: any;
+  context?: HttpContext;
+  observe?: 'body' | 'events' | 'response';
```

Comment:
Will also cause typescript error. For the get you cannot have responseType 'json' and observe 'events'. It's why that was any because I didn't found a good solution for that.